src/main/java/sirius/biz/tenants/Tenants.java

```diff
@@ -284,4 +284,60 @@ public String fetchCachedTenantName(I tenantId) {
     public String fetchCachedTenantName(BaseEntityRef<I, T> tenantRef) {
         return fetchCachedTenant(tenantRef).map(tenant -> tenant.getTenantData().getName()).orElse("");
     }
+
+    /**
+     * Provides access to the user account with the given id
+     * <p>
+     * This utilizes the cache maintained by the {@link TenantUserManager} and is therefore quite efficient
+     *
+     * @param userId the id of the user to fetch
+     * @return the user account with the given id or an empty optional if the user cannot be resolved
+     */
+    public Optional<U> fetchCachedUserAccount(I userId) {
```

Comment:
use a String here - also change for fetchCachedTenant - the lookup is based on strings anyway and this makes it easier when working with generic code.